### PR TITLE
fix: empty stack trace being logged to Kibana

### PR DESCRIPTION
**Pull request by MitchelPaulin**
This is hot fixing a fix to the logging for stack traces in 20.21.7 to hopefully get the stack trace info asap.

### CODE REVIEW

File: editor/d2l-rubric-structure-editor.js [hunk 1]
```diff
@@ -369,7 +369,7 @@ Polymer({
 			}.bind(this)).catch(function(err) {
 				this.dispatchEvent(new CustomEvent('d2l-rubric-editor-save-error', {
 					detail: {
-						message: err.messagem
```
Comment (MitchelPaulin) [inline editor/d2l-rubric-structure-editor.js:372]: Unrelated to the stack trace problem but a typo I found when updating this file

**Review by devpow112 — APPROVED** [review]

**Review by mpharoah-d2l — DISMISSED** [review]

**Comment by devpow112** [comment]
This is pre auto release so the release creation is manual. We should probably follow he old convention for consistency. So this would be `v3.13.8-20.21.7-hotfix-3`

**Comment by MitchelPaulin** [comment]
> 
> 
> This is pre auto release so the release creation is manual. We should probably follow the old convention for consistency. So this would be `v3.13.8-20.21.7-hotfix-3`.

As in this should be the name of the branch?

**Comment by devpow112** [comment]
> > This is pre auto release so the release creation is manual. We should probably follow the old convention for consistency. So this would be `v3.13.8-20.21.7-hotfix-3`.
> 
> As in this should be the name of the branch?

Sorry wasn't clear. That should be the tag name when you manually create the release.

**Comment by devpow112** [comment]
Don't forget to do 20.21.8 as well which is the `3.18.x` branch.

**Pull request closed without merging**